Jenkinsfile

```diff
@@ -262,4 +256,34 @@ EOF
     }
   }
 
+}
+
+def buildProject(targetPlatform, forceLocalDeployment = false) {
+  withEnv(["TARGET_PLATFORM=$targetPlatform", "FORCE_LOCAL_DEPLOYMENT=$forceLocalDeployment"]) {
+    sh '''
+      GOALS='clean javadoc:aggregate-jar test deploy'
```

Comment:
Small nit: I'd do verify instead of test

Reply:
I didn't touch the previous goals but makes sense - fixed!